### ISSUE TITLE
wcs2:req21 & wcs2:req22 are failed with validation error if it has the <nilValues> element

### ISSUE THREAD

**Issue by keshav-nangare** [opengeospatial/ets-wcs20#77]
error log: 

```
Request d8972e1957_1:
   Method: get
   URL: http://34.207.148.210:6080/arcgis/services/wcs111/ImageServer/WCSServer?service=WCS&VERSION=2.0.1&request=DescribeCoverage&CoverageId=Coverage1
   Response from parser ctlp:XMLValidatingParser:
      
   Messages from parser ctlp:XMLValidatingParser:
        Validation error:
  cvc-complex-type.4: Attribute 'id' must appear on element 'wcs:CoverageDescription'.
Validation error:
  cvc-complex-type.2.4.a: Invalid content was found starting with element 'swe:nilValues'. One of '{"http://www.opengis.net/swe/2.0":constraint, "http://www.opengis.net/swe/2.0":value}' is expected.
Validation error:
  cvc-complex-type.2.4.a: Invalid content was found starting with element 'swe:nilValues'. One of '{"http://www.opengis.net/swe/2.0":constraint, "http://www.opengis.net/swe/2.0":value}' is expected.
Validation error:
  cvc-complex-type.2.4.a: Invalid content was found starting with element 'swe:nilValues'. One of '{"http://www.opengis.net/swe/2.0":constraint, "http://www.opengis.net/swe/2.0":value}' is expected.
4 validation errors detected.
```

**Comment by dstenger** [comment]
From mailing list:

> Speaking about wcs 2.0.1, I kind of bypassed another problem that happens when there is “nodata” pixel inside the coverage. The spec is using “nilvalue” to tell clients what is the nodata pixel values.
> 
>  
> 
> Spec is as follows as I found online: http://schemas.opengis.net/sweCommon/2.0/examples/spec/nilValues.xml
> 
>  
> 
> Our service you can used to see how we implemented is like this: [ENDPOINT]
> 
>  
> 
> Teamengine for wcs 2.0 core has test: req21,22. They are telling me my service has validation err:
> 
> Validation error:
> 
>   cvc-complex-type.2.4.a: Invalid content was found starting with element 'swe:nilValues'. One of '{"http://www.opengis.net/swe/2.0":constraint, "http://www.opengis.net/swe/2.0":value}' is expected.
> 
>  
> 
> Can you advise how we should implement this part, our client-side products need this value to draw nodata pixels and we don’t want it violates specs too.

**Comment by lgoltz** [comment]
The version of the SWE schemas is outdated:

Currently 2.0.0 is included: 
https://github.com/opengeospatial/ets-wcs20/blob/80031ab5ef7233b58468ba1977b40e62bcc0b30c/src/main/resources/swe/2.0/swe.xsd#L1

The current version is 2.0.1: http://schemas.opengis.net/sweCommon/2.0/swe.xsd

**Comment by lgoltz** [comment]
As the service is not available PR #82 was not tested yet.
